_posts/2025-11-12-homebrew-5.0.0.md

```diff
@@ -0,0 +1,84 @@
+- [`launchctl` service removal `sudo` failures are non-fatal.](https://github.com/Homebrew/brew/pull/20803)
+- [`CGO_ENABLED` is enabled by default on ARM64 Linux.](https://github.com/Homebrew/brew/pull/20839)
+- [`pkgconf` will be reinstalled automatically on a macOS version mismatch.](https://github.com/Homebrew/brew/pull/20674)
+- [`rubydoc.brew.sh` has been redirected to `docs.brew.sh/rubydoc`](https://github.com/Homebrew/brew/pull/20884)
+- [`head` Git URLs always require a branch name.](https://github.com/Homebrew/brew/pull/20402)
+- [`HOMEBREW_FORBIDDEN_CASK_ARTIFACTS` allows selectively blocking cask install methods.](https://github.com/Homebrew/brew/pull/20806)
+- [`HOMEBREW_DEVELOPER` is required when installing formulae from paths.](https://github.com/Homebrew/brew/pull/20414)
+- [Formula's `compatibility_version` DSL will help us reduce the number of upgrades needed in future.](https://github.com/Homebrew/brew/pull/20804)
+- [Formula's `no_linkage` DSL will help stricter linkage detection and avoid issues.](https://github.com/Homebrew/brew/pull/20807)
+- [Formulae can be migrated to casks within the same tap.](https://github.com/Homebrew/brew/pull/20800)
+- [Linux's host `libstdc++` (instead of `gcc`) is used for automatically determining if a brewed `gcc` dependency is needed.](https://github.com/Homebrew/brew/pull/20682)
+- [Homebrew will avoid downloading JSON API files multiple times per run.](https://github.com/Homebrew/brew/pull/20619)
+- [Homebrew no longer provides CI for macOS Ventura in Homebrew/core.](https://github.com/Homebrew/brew/pull/20672)
+- [Homebrew's Linux CI uses GCC 12.](https://github.com/Homebrew/brew/pull/20609)
+- [Homebrew will not run on macOS Mojave and older.](https://github.com/Homebrew/brew/pull/20681)
+- [Homebrew prefers LLVM's `clang` over `gcc` on macOS.](https://github.com/Homebrew/brew/pull/20696)
+- [Homebrew defaults to `-O2` with `gcc`.](https://github.com/Homebrew/brew/pull/20716)
+- [Dependencies are resolved correctly when installing older bottles.](https://github.com/Homebrew/brew/pull/20802)
+- [Casks are better supported and have better usability on Linux.](https://github.com/Homebrew/brew/pull/20925)
+
+Finally:
+
+- [We have clarified our stance on adult content in Homebrew formulae and casks.](https://github.com/Homebrew/brew/pull/20766)
+- [We have loosened our criteria for accepting software into Homebrew.](https://github.com/Homebrew/brew/pull/20981)
+- [Homebrew accepts donations through GitHub Sponsors](https://github.com/sponsors/Homebrew) and [still accepts donations through Patreon](https://www.patreon.com/homebrew).
+  If you can afford it, please consider donating.
+  If you'd rather not use GitHub Sponsors or Patreon (our preferred donation methods), [check out the other ways to donate in our README](https://github.com/Homebrew/brew/#donations).
```

Comment:
Is there a reason we don't mention donating via Open Collective as an option? We mention it in the linked `README` and (assuming I understand correctly) it seems like that would preferable compared to GitHub Sponsors and Patreon if those platforms have additional fees on top (i.e., before the money reaches Open Collective).

Reply:
No reason, will add.